scripts/mysql_install_db.sh

```diff
@@ -459,6 +460,19 @@ mysqld_install_cmd_line()
 }
 
 
+# Create database directories
+if test "$skip_install_test_db" -eq 0
+then
+  install_params="$install_params
+CREATE DATABASE test;"
+else
+  install_params="$install_params
+SET @skip_install_test_db=$skip_install_test_db;"
+fi
+
+install_params="$install_params
+CREATE DATABASE mysql;"
```

Comment:
Yes, this looks better. It'd look even better if it were moved into `mysql_system_tables.sql` with `IF NOT EXISTS` added.

Reply:
In a similar way I'd move `test` creation and set install params unconditionally.

Reply:
ack done.

Reply:
Nice, but why these lines are still here?

Reply:
rebase fail. fixed. sorry